tests/Tests.AzureAppConfiguration/FeatureManagementTests.cs

```diff
@@ -185,6 +185,21 @@ public class FeatureManagementTests
                 eTag: new ETag("0a76e3d7-7ec1-4e37-883c-9ea6d0d89e63"),
                 contentType: "text");
 
+        readonly ConfigurationSetting Feature_RequirementTypeAll = ConfigurationModelFactory.ConfigurationSetting(
+            key: FeatureManagementConstants.FeatureFlagMarker + "Feature_All",
+                value: @"
+                        {
+                          ""id"": ""Feature_All"",
+                          ""enabled"": true,
+                          ""conditions"": {
+                            ""requirement_type"": ""All"",
```

Comment:
Is `requirement_type` a property of `conditions`?

Reply:
We originally had `requirement_type` as part of `conditions` in #418 but it wasn't being returned by the SDK if it was in `conditions` so I moved it. This variable isn't used anywhere so I didn't catch that I forgot to change this, thanks.